src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -724,7 +728,40 @@ private void allergy(String personID, String encounterID, Entry allergy) throws
     Code coding = allergy.codes.get(0);
 
     s.append(coding.code).append(',');
-    s.append(clean(coding.display));
+    s.append(clean(coding.display)).append(',');
+
+    int reactionsSize = 0;
+    if (allergy.reactions != null) {
+      reactionsSize = allergy.reactions.size();
+    }
+    Function<Map.Entry<HealthRecord.Code, HealthRecord.ReactionSeverity>, String> template =
+        mapEntry -> {
+          StringBuilder reactionBuilder = new StringBuilder();
+          reactionBuilder.append(mapEntry.getKey().code).append(',');
+          reactionBuilder.append(mapEntry.getValue());
+          return reactionBuilder.toString();
+        };
+
+    switch (reactionsSize) {
+      case 0:
+        s.append(",,,");
+        break;
+      case 1:
+        s.append(allergy.reactions.entrySet().stream().map(template).collect(Collectors.joining()));
+        s.append(",,");
+        break;
+      case 2:
+        s.append(allergy.reactions.entrySet().stream().map(template)
```

Comment:
We could remove this case and fall back to default if there are 2 reactions, to ensure that reaction1 is always the more severe of the 2. Not sure if that matters.